src/hardware/serialport/softmodem.cpp

```diff
@@ -250,7 +250,7 @@ void CSerialModem::SendNumber(uint32_t val)
 }
 
 void CSerialModem::SendRes(const ResTypes response) {
-	char const * string = nullptr;
+	const char* string = nullptr;
```

Comment:
(this `string` variable name is pretty bad.. like calling a variable `int` :grimacing: :laughing: !)

This function sends either a number-code or text response, based on the incoming response-type enum. This `string` variable is only used to hold the text response (in that code path). 

Maybe rename the variable (or eliminate the need for it using a lamda that returns the literal char*)?

Reply:
Renamed the variable. Refactoring would be more complicated (switch/case sets more than just the string variable), I prefer not to introduce such a risk.

Reply:
Makes perfect sense @FeralChild64 :+1: